app/components/dashboard-item/template.hbs

```diff
@@ -1,22 +1,22 @@
 <a href="{{node.links.html}}"
    onclick={{action 'click' 'link' 'Dashboard - visit_node' target=analytics}}
 >
-    <div class="m-v-sm DashboardItem">
+    <div class="m-v-sm {{local-class 'DashboardItem'}}">
```

Comment:
You sometimes use `class="... {{local-class ...}}"` and sometimes `class="..." local-class="..."`. Is there a benefit one way or the other?

`local-class="..."` seems a bit cleaner to me, and my understanding is we could use it almost everywhere.

Reply:
I agree, it is cleaner, but it only works on transformed class names, so global classes won't work. E.g. the `m-v-sm` for bootstrap.

Reply:
Right, so
```hbs
<div class="m-v-sm" local-class="DashboardItem">
```
vs
```hbs
<div class="m-v-sm {{local-class 'DashboardItem'}}">
```
Wondering if the latter has any benefit.

Reply:
There's no difference AFAICT